index.html

```diff
@@ -1735,22 +1735,23 @@
   <p>
     <span class="practicelab" id="abuse-reporting">
       Systems that allow for communicating on the web must provide an
-      effective capability to report abuse.
-        </span>
+      effective capability to report [=abuse=].
+    </span>
   </p>
 </div>
 <div class="practice" data-audiences="websites user-agents api-designers">
   <p>
     <span class="practicelab" id="abuse-protection">
       [=User agents=] and [=sites=] must
-      take steps to protect their users from abusive behaviour, and abuse
+      take steps to protect their users from abusive behaviour, and [=abuse=]
       mitigation must be considered when designing web platform features.
     </span>
   </p>
 </div>
 
-Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
-through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the web,
+Digital <dfn>abuse</dfn> is the cruel treatment of a person through digital means. Online <dfn>harassment</dfn> 
```

Comment:
```suggestion
Digital <dfn>abuse</dfn> is the mistreatment of a person through digital means. Online <dfn>harassment</dfn> 
```

Reply:
Slightly broader than "cruel" is useful because some annoyances like spam can be mistreatment, but we don't think of them as cruel or as necessarily harassment.